### PR TITLE
feature/fate320269: Apache2 chapter improvements

### PR DESCRIPTION
**Pull request by Vogtinator**

### CODE REVIEW

File: xml/apache2.xml [hunk 1]
```diff
@@ -2161,8 +2161,8 @@ sudo cp new.cert.key /etc/apache2/ssl.key/server.key</screen>
      have an officially signed certificate.
     </para>
     <para>
-     The best-known commercial CAs are Comodo
-     (<link xlink:href="https://www.comodo.com"/>) and Symantec
+     The best-known commercial CAs are Thawte
```
Comment (sknorr) [inline xml/apache2.xml]: Thawte is now Symantec (even their logo is "Thawte from Symantec" now). Also, Comodo is now the largest CA, I think. How about adding GoDaddy or DigiCert too?

Comment (Vogtinator) [inline xml/apache2.xml]: > Also, Comodo is now the largest CA, I think.

https://letsencrypt.org//2016/06/23/defending-our-brand.html

> How about adding GoDaddy or DigiCert too?

I'd be fine with those.

Comment (Vogtinator) [inline xml/apache2.xml]: @keichwa agrees with me that that sentence should be removed completely

**Comment by fsundermeyer** [comment]
Hanx!

**Pull request merged**